signup-page.html

```diff
@@ -0,0 +1,111 @@
+</head>
+<style>
+    .main-container{
+        background-color: #EDF0F5;
+        padding: 1% 3%;
+        margin: 3% 35%;
+        font-size: 18px;
+        font-family:Georgia, 'Times New Roman', Times, serif;
+            }
+    .btn-singup{
+        background-color: #5E9B4D;
+        margin: 20px 0px;
+        padding: 10px 50px;
+        border: 2px solid #5E9B4D;
+        border-radius: 5px;
+        font-weight: bold;
+        font-size: 20px;
+        color: white;
+    }
+    .textStyle{
+        font-weight: bold;
+        color: #BEC3C9;
+      
+    }
+    .textInput{
+        width: 40%;
+        height: 35px;
+        border-radius: 5px;
+        margin: 5px 0px;
+        font-size: 20px;
+        border:none;
+        padding: 1% 3%;
+        }
+    .inputMobileAndPassword{
+        width: 90%;
+    }
+    select{
+        width: 12%;
+        height: 30px;
+        margin: 0px 0px;
+        padding: 0px;
+    }
+    .termsCondition{
+        width: 80%;
+        font-size: 12px;
+    }
+   
+    .month{
+        margin-top: .5em;
+    }
+    .gender{
+        margin-top: .5em;
+    }
+    .genderClass {
+        margin: 1em 1em ;
+
+    }
+</style>
+<body>
+    <div class ="main-container">
+        <h1 class="textBold">Sign Up </h1>
+        <h3 class="textStyle">It's quick and easy.</h3>
+        <form action="">
+            <input type="text" class="textInput" name="firstname" placeholder="First name"> 
+            <input type="text" class="textInput" name ="lastname" style="margin-left: 10px;" placeholder="Last name"> <br>
+            <input type="text" class="textInput inputMobileAndPassword" name ="mobileoremail" placeholder="Mobile number or email"><br>
+            <input type="text" class="textInput inputMobileAndPassword" name ="password" placeholder="New password"><br>
+
+            <label for="birthday" class="birthday textStyle">Birthday</label> <br>
+            <select name="month" class="month" id="">
+                <option value="jan">Jan</option>
+                <option value="feb">Feb</option>
+                <option value="mar">Mar</option>
+                <option value="apr">Apr</option>
+            </select>
+            <select name="date" id="">
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+                <option value="6">6</option>
+            </select>
+            <select name="year" id="">
+                <option value="2000">2000</option>
+                <option value="2001">2001</option>
+                <option value="2002">2002</option>
+                <option value="2003">2003</option>
+                <option value="2004">2004</option>
+                <option value="2005">2005</option>
+            </select><br>
+
+            <label for="gender" class="gender textStyle">Gender</label><br>
```

Comment:
`textStyle` cannot be customized classname, as it can be used as an inline css style in React code, so it's kind of a keyword.